lib/core/utils/dq-element.js

```diff
@@ -39,8 +39,12 @@ function DqElement(element, options, spec) {
    * The generated HTML source code of the element
    * @type {String}
    */
-  this.source =
-    this.spec.source !== undefined ? this.spec.source : getSource(element);
+  // TODO: es-moduels_audit
+  this.source = axe._audit.noHtml
+    ? ''
```

Comment:
Should this be undefined?

Reply:
`undefined` or `null`? IMO `null` is preferred, since it's explicitly not set. NBD tho :)